src/util/numbering.h

```diff
@@ -129,15 +129,15 @@ class hash_numbering final
     return (result.first)->second;
   }
 
-  bool get_number(const T &a, number_type &n) const
+  optionalt<number_type> get_number(const T &a) const
   {
-    typename numberst::const_iterator it=numbers.find(a);
+    const auto it = numbers.find(a);
 
-    if(it==numbers.end())
-      return true;
-
-    n=it->second;
-    return false;
+    if(it == numbers.end())
+    {
```

Comment:
Braces not really necessary here (and other places)

Reply:
[Braces are better](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/)